### DISCUSSION TITLE
Some of the variables from the Player class is not available

### DISCUSSION THREAD

**Discussion by barbasan** [cwendt94/espn-api#303] [Q&A] [answered]
For player class bastbetball total_points, avg_points, projected_total_points, projected_avg_points are not available.

AttributeError: 'Player' object has no attribute 'avg_points'

**Comment by cwendt94** [comment, marked as answer]
They should be available what version of the package are you running?

**Reply by barbasan** [reply]
0.19.0

**Reply by barbasan** [reply]
Now I realize that I could not fetch points with .stats[xx202x] too. This was not the case when I start this discussion. As you suspected, the version of the package is the problem. Although I have it upgraded that day and restarted the session, the problem somehow persisted. It is working fine now! Thank you for creating and developing this work

**Reply by cwendt94** [reply]
Glad it is working now! That is strange!
